src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figuresArray = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figuresArray.length; i++) {
+            if (i > Math.floor(figuresArray.length / 2)) {
+                figuresArray[i] = figureSupplier.getRandomFigure(100);
```

Comment:
Passing a magic number '100' directly to the `getRandomFigure` method is not recommended. Consider defining it as a constant at the class level or passing it through the constructor of `FigureSupplier`.